### DISCUSSION TITLE
[월드 1.1.5] UnityEvent$1<Vector3>

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#561] [Issue Report (Legacy)]
월드 버전 : 1.1.5 

이벤트 변수 정의
`public attackEvent2 : UnityEvent$1<Vector3>;`
인스턴스화
`this.attackEvent2 = new UnityEvent$1<Vector3>();`
호출에서 문제 없었으나 
`this.attackEvent2.Invoke(this.transform.position);`

이벤트를 등록하면 다음과 같이 **런타임 에러**가 나는것을 확인 했습니다 

` zombie.attackEvent2.AddListener((pos)=>{
                console.log('attack2: ' + pos);
            });`

![image](https://user-images.githubusercontent.com/50188795/160992737-c5b7b2ac-84e5-43d2-be88-d1147678f193.png)

다음과 같이 두 케이스는 정상 동작 했습니다 
![image](https://user-images.githubusercontent.com/50188795/160992114-4f73e629-dd5c-491f-812e-4f5d42c68f07.png)
![image](https://user-images.githubusercontent.com/50188795/160992253-3b3ac196-a36c-4924-b259-6bf7e375245d.png)
![image](https://user-images.githubusercontent.com/50188795/160992274-b5f147e3-1d67-474c-ba94-aaa8bbed858b.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 월드 1.1.7 버전이 업데이트 되었습니다.
업데이트 이후 같은 증상이 나타나는지 확인 부탁 드립니다.
https://studio.zepeto.me/kr/guides/world-117
